src/calculateAnimals.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const calculateAnimals = (generatedNumber, userNumber) => {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < generatedNumber.length; i++) {
```

Comment:
Here we can use `reduce()` for counting bulls and cows. You leave your data as string to use includes(), but in array we have this method too. Try to do it with reduce().